flexible/java-11/analytics/src/main/java/com/example/analytics/Main.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.analytics;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.ServletComponentScan;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@SpringBootApplication
+@ServletComponentScan("com.example.analytics")
```

Comment:
It's very cool that Spring boot can run servlets, but do we expect customers to do this or is this part of a migration story? It would be more idiomatic to have a full spring boot app.

Reply:
Yeah it's cool that SpringBoot can run servlets!! I am unaware of the migration story here. Also, yes we expect the customers to do this. Essentially they can use either springboot or jetty for this purpose.